issuer_controller/src/issuer.py

```diff
@@ -362,19 +442,78 @@ def log_timing_method(method, start_time, end_time, success, data=None):
         timing_lock.release()
 
 
+def log_timing_event(method, message, start_time, end_time, success, outcome=None):
+    """Record a timing event in the system log or http endpoint."""
+
+    if (not TRACE_EVENTS) and (not message.get("trace")):
+        return
+    if not TRACE_TARGET:
+        return
```

Comment:
Would it make sense to log a `warning` here, explaining why timings are not running?

Based on my understanding:  if this function is being called regardless of whether timing is enabled or not (and just returning here) we could check whether timing is enabled at all as first condition and return without warning if it isn't, and only log warnings in these scenarios (which would assume we want timings, but we didn't set them up right).